src/components/pages/matchTrial/cardMatch.js

```diff
@@ -0,0 +1,113 @@
+import Calendar from '../../common/icons/Calendar';
+import Tick from '../../common/icons/Tick';
+import TestTube from '../../common/icons/TestTube';
+import Plus from '../../common/icons/Plus';
+import Marker from '../../common/icons/Marker';
+
+import * as S from './style';
+
+const matchCard = ({ windowWidth }) => {
+  return (
+    <S.Wrapper>
+      <S.HeadSection>
+        <div>
+          <S.PrimarySpam>NCT Number: </S.PrimarySpam>
+          <span>NCT0355514</span>
+        </div>
+        <S.PrimarySpam style={{ fontWeight: 'bold' }}>PRONOUNCE</S.PrimarySpam>
+      </S.HeadSection>
+      <S.HeadPragraph>
+        A Trial Comparing Cardiovascular Safety of Degarelix Versus Leuprolide
+        in Patients With Advanced Prostate Cancer and Cardiovascular Disease
+      </S.HeadPragraph>
+      <S.RowSection>
+        <S.ColumnSection>
+          <S.FieldWrapper>
+            <div>
+              <Calendar></Calendar>
+            </div>
+            <div>
+              <span>Starting Date: Sept 27, 2018</span>
+              <span>Finish Date: Ongoing</span>
+            </div>
+          </S.FieldWrapper>
+        </S.ColumnSection>
+        <S.ColumnSection>
+          <S.FieldWrapper>
+            <div>
+              <Avatar></Avatar>
+            </div>
+            <span>Phase: 3</span>
+          </S.FieldWrapper>
+          <S.FieldWrapper>
+            <div>
+              <Tick></Tick>
+            </div>
+            <span>Recruiting</span>
+          </S.FieldWrapper>
+        </S.ColumnSection>
+        <S.FieldWrapper>
+          <div>
+            <Avatar></Avatar>
+          </div>
+          <span>Enrolled: 6,000,000</span>
+        </S.FieldWrapper>
+      </S.RowSection>
+      <S.RowSection>
+        <S.FieldWrapper>
+          <div>
+            <TestTube></TestTube>
+          </div>
+          <span>Interventions: Degarelix, Leuprolide</span>
+        </S.FieldWrapper>
+        <S.FieldWrapper style={{ gridColumn: '2/4' }}>
+          <div>
+            <Plus></Plus>
+          </div>
+          <span>
+            Sponsor/Collaborators: Ferring Pharmaceuticals, Memorial Sloan
+            Kettering
+          </span>
+        </S.FieldWrapper>
+      </S.RowSection>
+      <S.FieldWrapper style={{ gridColumn: '2/4' }}>
+        <div>
+          <Plus></Plus>
+        </div>
+        <span>
+          Allocation: Randomized. Intervention Model: Paralle. Assignment.
+          Masking: Single (Outcomes Assessor). Primary Purpose: Treatment
+        </span>
+      </S.FieldWrapper>
+      <Table windowWidth={windowWidth} />
+      <section>
+        <S.BoldParagraph>Nearest Trial Locations: </S.BoldParagraph>
+        <S.LocationSection>
+          <span>
+            <Marker width="14px"></Marker>5 miles
+          </span>
+          <span>St Bartholomew’s Hospital, London EC1A 7JQ</span>
+        </S.LocationSection>
+        <S.BottomSection>
+          <span style={{ gridColumn: '1 / 3' }}>
```

Comment:
This looks like a clickable link, and therefore should be an anchor link. I believe there is a pre-made anchor link defined in `Buttons.js` for this.

<img width="307" alt="Screenshot 2019-10-13 at 11 21 46" src="https://user-images.githubusercontent.com/28146666/66714688-af722b00-edb1-11e9-912f-9a735dbec610.png">